.github/workflows/local_example.yml

```diff
@@ -17,11 +18,21 @@ jobs:
 
     - name: Get dependencies
       run: |
-        sudo apt-get install -y mysql-server mysql-client make unzip g++ etcd curl git wget
+        if [ ${{matrix.TARGET}} = "ubuntu-latest"]; then
+          sudo apt-get install -y mysql-server mysql-client make unzip g++ etcd curl git wget
+          sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
+          sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
+        elif [ ${{matrix.TARGET}} = "macos-latest" ]; then
+          /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+          brew install mysql@5.7 make unzip etcd curl git wget
+        elif [ ${{matrix.TARGET}} = "centos-latest" ]; then
+          yum update 
+          sudo yum install mysql-community-server mysql-client unzip g++ etcd curl git wget
+        else exit 1;
+        fi
         sudo service mysql stop
         sudo service etcd stop
```

Comment:
This will need to be part of the matrix too, since macOS won't use it. yum packages also won't auto-start daemons, that is a debianism.